.github/workflows/main.yml

```diff
@@ -100,4 +100,44 @@ jobs:
       - name: Run tests
         run: |
           source ${{ github.workspace }}/venv/bin/activate
-          make test
+          make test-py
+
+  build-js:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
```

Comment:
What's the relevance of this comment?

Reply:
Copy/pasta from GitHub's docs… which is now wrong isn't it 😬